fl4health/reporting/wandb_reporter.py

```diff
@@ -110,12 +116,12 @@ def define_metrics(self) -> None:
         self.run.define_metric("round_end", summary="none", hidden=True)
         # A server round contains a fit_round and maybe also an evaluate round
         self.run.define_metric("fit_round_start", summary="none", hidden=True)
-        self.run.define_metric("fit_round_time_elapsed", summary="none", hidden=True)
         self.run.define_metric("fit_round_end", summary="none", hidden=True)
         self.run.define_metric("eval_round_start", summary="none", hidden=True)
-        self.run.define_metric("eval_round_time_elapsed", summary="none", hidden=True)
         self.run.define_metric("eval_round_end", summary="none", hidden=True)
         # The metrics computed on all the samples from the final epoch, or the entire round if training by steps
+        self.run.define_metric("fit_round_time_elapsed", summary="none")
```

Comment:
Ok sorry I hadn't read your previous comment yet. As I understand it, all that hidden does is hides the metric from the automatically generated plots. Since fit_round_time_elapsed and eval_round_time_elapsed are both datetime strings, they can't be plotted anyways. I originally also thought it would be a good idea to view the elapsed time plotted against the round on x-axis, but that would involve changing their type and I kind of just pushed off the idea for a future PR. That being said if you want to implement that here I'm cool with that.

Reply:
Sounds good, will move it over to be an int (duration in seconds)